meorg_client/client.py

```diff
@@ -64,13 +64,16 @@ def _make_request(self, method: str, endpoint: str, data: dict=None, headers: di
 
         # GET/PUT requests have the data interpolated into the url
         if method in ['GET', 'PUT']:
-            endpoint = endpoint.format(data)
+            endpoint = endpoint.format(**data)
+
+        # Remove the leading slash
+        endpoint = endpoint[1:] if endpoint[0] == '/' else endpoint
```

Comment:
There is probably a more elegant way to do this, essentially an os.path.join, but for URLS.

Come to think of it, the os.path.join might actually work for this?